test/server_test.py

```diff
@@ -592,6 +601,24 @@ def passwordChangeSuccess():
 			user = User.get_by_name(test_name)
 			assert_that(user.pass_hash, equal_to(new_hash))
 
+		@it('User notified by email')
+		@patch('util.sendEmail')
+		def passwordChangeNotifiesByEmail(mock_emailer):
```

Comment:
My first instinct was to tell you to combine this with the previous test for duplicating success verification, but on second thought I like that you split password change verification and email sent verification into two separate tests. That way a failure in one won't affect the other.